fl4health/utils/functions.py

```diff
@@ -1,36 +1,8 @@
-from logging import WARNING
-from typing import Tuple
-
 import torch.nn as nn
-from flwr.common.logger import log
 from flwr.common.parameter import ndarrays_to_parameters
 from flwr.common.typing import Parameters
-from opacus.validators import ModuleValidator
 
 
 def get_all_model_parameters(model: nn.Module) -> Parameters:
     # Extracting all model parameters and converting to Parameters object
     return ndarrays_to_parameters([val.cpu().numpy() for _, val in model.state_dict().items()])
-
```

Comment:
Moved this functionality into its own file (fl4health/utils/privacy_utilities.py)